src/Database/Schema/SqlserverSchema.php

```diff
@@ -12,26 +13,47 @@
  * @since         3.0.0
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
+/**
+ * 
+ * UPDATED VERSION TO WORK WITH CAKEPHP AND MSSQL VIEWS.
+ * 
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
 namespace Cake\Database\Schema;
 
+use Cake\Database\Schema\Table;
+
 /**
  * Schema management/reflection features for SQLServer.
  */
-class SqlserverSchema extends BaseSchema
-{
+class SqlserverSchema extends BaseSchema {
```

Comment:
If you use PhpStorm (which I presume you do as it has a tendency to adapt all code to your editing style), I suggest you use "PSR-2" code style preset *in project scope* as it closely matches Cake's coding standard and then Code->Reformat in order to bring everything up to new coding style
![screenshot_1](https://cloud.githubusercontent.com/assets/2865341/11786479/865ccc50-a28f-11e5-9cf4-8960b2469ba5.png)

If you don't use PhpStorm, forgive me for off-topic tip